### DISCUSSION TITLE
Can't modify RetroArch config

### DISCUSSION THREAD

**Discussion by khromov** [eduardofilo/RG350_adam_image#100] [RetroArch topics]
When I go to Configuration file > Save current configuration after modifying some RetroArch settings, an error is displayed: `Failed saving config to  "/media/data/local/home/.ret....`.  This makes it impossible to reconfigure RetroArch.

I'm trying to change the video filtering from nearest neighbor to bicubic for SNES games if that helps.

**Comment by eduardofilo** [comment]
This error occurs when you try to save changes to that file with some content (ROM) loaded. To make changes to this file, you have to open RetroArch from the independent launcher found in `GMenu2X > Emulators`. In the next version of the image I will include a launcher in SimpleMenu. In any case, the changes in the specific systems are not made from the general configuration but through overrides. Go over FAQ number 3 to see if it helps you:
https://github.com/eduardofilo/RG350_adam_image#faq

**Comment by SuperBadger77** [comment]
@eduardofilo how close are we to the v1.3 release?

**Comment by eduardofilo** [comment]
I was about to take it out last weekend, but new problems/requests appeared. Let's see if this weekend I can close it.

**Comment by eduardofilo** [comment]
@khromov I have created the FAQ number 6 specifically for your question:

https://github.com/eduardofilo/RG350_adam_image#faq

**Comment by khromov** [comment]
@eduardofilo Thanks for your help Eduardo. I tried following the FAQ ("Close content" > Modify settings > Save current configuration".) The configuration saves fine, but when I load up a SNES game after closing RetroArch, the config is still using "nearest neighbor" even though I changed the general config.

Is there some SNES-specific configuration file that overrides the general one? How would I modify that?

PS. I tried the second way of opening the RetroArch launcher through `GMenu2X` as well, but I don't have a a `GMenu2X` option anywhere. I tried looking through Commander app but I can't find the OPK anywhere for it.

**Comment by eduardofilo** [comment]
Yes, the SNES and almost all systems have overrides, i.e. specific configurations above the general conf. You can read about that concept in FAQ number 3.

The RetroArch launcher in GMenu2X is in `emulators` section.
